CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Support for differentiation with respect to `ComplexPolySlab.vertices`.
 - Introduce RF material library. Users can now export `rf_material_library` from `tidy3d.plugins.microwave`.
+- Users can now export `SimulationData` to MATLAB `.mat` files with the `to_mat_file` method.
+- Introduce RF material library. Users can now import `rf_material_library` from `tidy3d.plugins.microwave`.
```

Comment:
This seems messed up

Reply:
I changed "import" from "export" in line 14 and messed this up. I will revert these changes.